docs/content/querying/segmentmetadataquery.md

```diff
@@ -29,6 +29,7 @@ There are several main parts to a segment metadata query:
 |toInclude|A JSON Object representing what columns should be included in the result. Defaults to "all".|no|
 |merge|Merge all individual segment metadata results into a single result|no|
 |context|See [Context](../querying/query-context.html)|no|
+|analysisTypes|A list of Strings specifying what column properties (e.g. cardinality, size) should be calculated and returned in the result. Defaults to ["CARDINALITY", "SIZE"]. If cardinality and/or size values are not needed, omitting those properties will result in a more efficient query.|no|
```

Comment:
I would prefer if we had consistent casing in our query APIs and default everything to lower / camelCase. Maybe we can do something to have better serde for enums?